### PR TITLE
2) Made software_versions optional

### LINKED ISSUE #171: Remove `multiqc` and `software_versions.yml` from rnadeseq report
### Description of feature

We just discussed with @qbicStefanC that it might make sense to already remove the `MultiQC`RNAseq part from the rnadeseq report so that it can be run with data coming from other projects.

### PR DESCRIPTION
**Pull request by WackerO**
This PR makes the software_versions an optional parameter, this will close #171 

## PR checklist

- [x] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
  - [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/rnadeseq/tree/master/.github/CONTRIBUTING.md)
  - [ ] If necessary, also make a PR on the nf-core/rnadeseq _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [x] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [x] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Comment by WackerO** [comment]
In 1) Scaling heatmaps, the container changes. After it is merged, I have to change the rsem 5sums in this PR

File: assets/RNAseq_report.Rmd [hunk 1]
```diff
@@ -1968,12 +1968,15 @@ quote_string <- ifelse(
     paste0("The results for all work packages (no quote/offer given to describe them) can be found in this report.")
 )
 cat(quote_string)
+
+# Initiate var in case no software_versions were provided
+is_csv <- F
```
Comment (ggabernet) [inline assets/RNAseq_report.Rmd:1973]: Could you give it an easier to understand name? e.g. `software_versions_csv` ?

**Review by ggabernet — APPROVED** [review]
Just a comment, otherwise looks good to me

File: assets/RNAseq_report.Rmd (hunk 1 above)
Comment (WackerO) [reply assets/RNAseq_report.Rmd:1973]: done!

**Pull request merged**